modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -13,7 +13,8 @@
           "task_config": {
             "buckets": "{$.meta.buckets}",
             "fileBucketMaps": "{$.meta.collection.files}",
-            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}"
+            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
+            "orcaDefaultRecoveryTypeOverride": "{$.meta.collection.meta.orcaDefaultRecoveryTypeOverride}"
```

Comment:
Wrong lambda

Reply:
the workflow should be `OrcaRecoveryWorkflow` or am I missing something?

Reply:
Yes, but this is not the right lambda. This is the ExtractFilepaths lambda. RequestFiles is further down.

Reply:
oops fixed.